_posts/2018-02-27-what-should-go-stdlib.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "What Should Go Into the C++ Standard Library"
+layout: blog
+sidenav: side-nav-blog.html
+published: false
```

Comment:
Since this is not future-dated, just set this to "true"

Reply:
Done